src/panels/config/core/ha-config-system-navigation.ts

```diff
@@ -50,57 +50,39 @@ class HaConfigSystemNavigation extends LitElement {
     const pages = configSections.general
       .filter((page) => canShowPage(this.hass, page))
       .map((page) => {
-        let description = "";
-
-        switch (page.translationKey) {
```

Comment:
Why remove the switch? The code is unreadable now.

Reply:
While I personally hate switch and prefer anything else, whether ternaries or if statements, I reverted this back to a switch statement to prevent this PR from going out of scope